### ISSUE TITLE
HomeKit Device integration recommended as a "fully functional" alternative

### ISSUE THREAD

**Issue by BillWiegert** [home-assistant/home-assistant.github.io#35894]
### Feedback

The warning blurb on this page recommends the HomeKit Device integration as a "fully functional" alternative now that ecobee developer accounts can no longer be created. I think this phrasing is misleading because the HomeKit integration lacks many features that would be of interest to a Home Assistant user.

Here are a few limitations of the HomeKit integration compared to the ecobee integration:

- No control over auxiliary heat
- No list of currently running equipment (can't tell when aux heat is running)
- No information from air quality sensors
- No support for vacation features

The HomeKit Device integration is certainly better than nothing, but its functionality is incomplete. As such I'd recommend changing the last sentence of the warning message to the following:

> In the meantime, you can use the [HomeKit Device](https://www.home-assistant.io/integrations/homekit_controller/) integration as an alternative that provides basic functionality.

### URL

https://www.home-assistant.io/integrations/ecobee/

### Version

2024.11.2

**Comment by f00kie** [comment]
I agree with this. I found it quite confusing and time consuming looking for ways to get to the above -- this thread is the only way I've been able to tell that you cannot do all the 'ecobee' things via the 'HomeKit Device' addon.
